### PR TITLE
feat: Configure contributions application registry category - EXO-61480 - Meeds-io/MPIs#37

### PR DESCRIPTION
**Pull request by sofyenne**
This change is going to add the contributions  application category

### CODE REVIEW

File: portlets/src/main/webapp/WEB-INF/conf/applicationRegistry/application-registry-configuration.xml [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <gnu.org/licenses>.
+-->
+
+<configuration
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
+        xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
+
+    <external-component-plugins>
```
Comment (azayati) [inline portlets/src/main/webapp/WEB-INF/conf/applicationRegistry/application-registry-configuration.xml]: indentation

**Review by azayati — CHANGES_REQUESTED** [review]

File: portlets/src/main/webapp/WEB-INF/conf/applicationRegistry/application-registry-configuration.xml (hunk 1 above)
Comment (sofyenne) [reply portlets/src/main/webapp/WEB-INF/conf/applicationRegistry/application-registry-configuration.xml]: done !

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**